beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/handler/ExpiredMetadataHandler.java

```diff
@@ -58,6 +60,26 @@ public ExpiredMetadataHandler(CleanerClientFactory cleanerClientFactory,
     return housekeepingMetadataRepository.findRecordsForCleanupByModifiedTimestamp(instant, pageable);
   }
 
+  @Override
+  public void handleDisabledTable(String database, String table) {
```

Comment:
This should then be renamed `disableTable` because here you are actually changing the status of a table in the backend to DISABLED.